src/libs/ReportUtils.js

```diff
@@ -1173,27 +1174,36 @@ function getPersonalDetailsForAccountID(accountID) {
  *
  * @param {Number} accountID
  * @param {Boolean} [shouldUseShortForm]
+ * @param {Boolean} shouldFallbackToHidden
  * @returns {String}
  */
-function getDisplayNameForParticipant(accountID, shouldUseShortForm = false) {
+function getDisplayNameForParticipant(accountID, shouldUseShortForm = false, shouldFallbackToHidden = true) {
     if (!accountID) {
         return '';
     }
     const personalDetails = getPersonalDetailsForAccountID(accountID);
+    // check if it's an invite account
```

Comment:
Let's make this comment more descriptive to explain why we are doing this, rather than just saying what we're doing.

Reply:
@puneetlath comment updated

Reply:
Great, thanks!